python/ray/data/tests/test_raydp_dataset.py

```diff
@@ -46,6 +46,13 @@ def test_raydp_to_spark(spark_on_ray_small):
     rows = [r.value for r in df.take(5)]
     assert values == rows
 
+def test_spark_on_ray(spark_on_ray_small):
+    spark = spark_on_ray_small
+    spark_df = spark.createDataFrame([(1, 0), (2, 0), (3, 1)], ["one", "two"])
+    ds = ray.data.from_spark(spark_df)
+    dataset = ds.to_torch(batch_size=2)
+    print(next(dataset.__iter__()))
```

Comment:
Instead of printing, could you `assert` the expected batch content here?

Reply:
Thanks, I have fixed it.